src/extensions/extensionsHelper.ts

```diff
@@ -393,18 +399,46 @@ export async function publishExtensionVersionFromLocalSource(
   );
   validateSpec(subbedSpec);
 
-  const consent = await confirmExtensionVersion(publisherId, extensionId, extensionSpec.version);
-  if (!consent) {
-    return;
-  }
-
   let extension;
   try {
     extension = await getExtension(`${publisherId}/${extensionId}`);
   } catch (err) {
     // Silently fail and continue the publish flow if extension not found.
   }
 
+  let notes: string;
+  try {
+    const changes = getLocalChangelog(rootDirectory);
+    notes = changes[extensionSpec.version];
+  } catch (err) {
+    throw new FirebaseError(
+      "No CHANGELOG.md file found. " +
+        "Please create one and add an entry for this version. " +
+        marked(
+          "See https://firebase.google.com/docs/extensions/alpha/create-user-docs#writing-changelog for more details."
```

Comment:
Note that these docs aren't yet published, but they will be by the time we release this.